alerts/machine-config-operator/MachineConfigDaemonPivotError.md

```diff
@@ -0,0 +1,151 @@
+# MCDPivotError
+
+## Meaning
+
+Alerts the User when the Machine Config Daemon
+is either unable to or detecs an error when
+trying to pivot OS image to a new version.
+
+>Alerts the user when an error is detected upon pivot.
+This triggers if the pivot errors are above zero for 2 minutes.
```

Comment:
Do we need these two sentences? They don't seem to add anything and the sentences before and after have better detail.

Reply:
I agree and went ahead and removed them.